browser/ui/brave_browser_command_controller.cc

```diff
@@ -430,11 +430,15 @@ void BraveBrowserCommandController::UpdateCommandForSplitView() {
   }
 
   UpdateCommandEnabled(IDC_NEW_SPLIT_VIEW, brave::CanOpenNewSplitViewForTab(
-                                               std::to_address(browser_)));
+                                               base::to_address(browser_)));
   UpdateCommandEnabled(IDC_TILE_TABS,
-                       brave::CanTileTabs(std::to_address(browser_)));
-  UpdateCommandEnabled(IDC_BREAK_TILE,
-                       brave::IsTabsTiled(std::to_address(browser_)));
+                       brave::CanTileTabs(base::to_address(browser_)));
+
+  const auto is_tab_tiled = brave::IsTabsTiled(base::to_address(browser_));
```

Comment:
A new banned feature: std::to_address

```
      std::to_address is banned because it is not guaranteed to be
      SFINAE-compatible. Use base::to_address from base/types/to_address.h
      instead.
```

Reply:
Filed an good first bug for those who are interested in contribution.

https://github.com/brave/brave-browser/issues/38658